xcmailr-webapp/src/main/java/controllers/restapi/util/Email.java

```diff
@@ -57,7 +56,7 @@ public EmailValidator(final Email dbIdAnnotation)
 
         public void validate(String value, String fieldName, Context context)
         {
-            String[] parts = StringUtils.split(StringUtils.defaultString(value), '@');
+            final String[] parts = HelperUtils.splitMailAddress(value);
```

Comment:
`HelperUtils.splitMailAddress()` may return null so we would have to handle null in the condition that follows.

Reply:
Nice catch. Fixed.